### DISCUSSION TITLE
Export read used in alignment

### DISCUSSION THREAD

**Discussion by racng** [milaboratory/mixcr#1809] [Q&A] [answered]
Suppose I ran mixcr align with a tag pattern like this: `(CELL:N{8})(UMI:N{8})gcgcg(R1:*)atatatat`
Is there a way to export the extracted R1 sequence and qualities? 
I tried `mixcr exportAlignments -f -targetSequences -targetQualities test.vdjca > test.alignments.tsv`. 
I wasn't sure if there is any additional processing was done to R1, and whether it is equivalent to "targetSequence" in this context.

**Comment by mizraelson** [comment, marked as answer]
`mixcr align` aligns the reads to the reference, therefore in the .vdjca file, you will only see the aligned reads. Depending on the preset you used, the file may contain only reads with a complete CDR3 sequence or may also include reads that do not cover the CDR3 but at least cover a part of a V or a J gene. Beyond that, no additional processing has been done to R1 at this stage.

**Reply by racng** [reply]
What if I used the `--write-all` tag during the `align` step. Would that let me export the parsed `(R1:*)` from aligned and unaligned reads when running `exportAlignments -targetSequences`?

**Reply by mizraelson** [reply]
No, everything that comes out of `mixcr align` has to be at least partially aligned to TCR or BCR sequence.
